docs/docs/guides/troubleshooting.mdx

```diff
@@ -102,6 +102,11 @@ After that make sure to run your package manager again, either `yarn` or `npm in
 
 **Solution:** Please upgrade to at least React Native 0.72 or downgrade to an older version of Reanimated 3.
 
+| Reanimated             | React-native |
+| ---------------------- | ------------ |
+| 3.1.0+, fabric enabled | 0.72.0+      |
+| 3.0.0+, fabric enabled | 0.71.0+      |
+
```

Comment:
But we haven't defined minimum React-Native for reanimated2, so this error won't trigger for Reanimated2, do you want to add it to docs for v2 anyway?

Reply:
I think that it looks better with one table since all the missing rows are "any"

Reply:
But this is not a true because any version has bottom and upper limit of RN version. For example, Reanimated 2.17.0 works with RN@0.64 and RN@0.71 but doesn't work with RN@0.72

Reply:
Are you sure? It seems that 2.17.0 works with RN@63-70, so it doesn't support RN@71 ...